mrp_partial_production/models/mrp.py

```diff
@@ -49,10 +49,11 @@ def get_qty_available_to_produce(self):
                                           res.quantity_done)
 
         for line, line_data in lines:
-            lines_dict[line.product_id.id] += line_data['qty'] / quantity
-
-        qty = [(result[key] / val) for key, val in lines_dict.items()
-               if val]
+            if line.product_id.type in ('service','consu'):
+                lines_dict.pop(line.product_id.id, None)
+            else:
```

Comment:
@frahikLV please remove the else statement, instead add a continue inside the if

Reply:
Fixed 👍